### PR TITLE
[FIX] edi: Fixing error on tests when trying to handle multi companies

### PR DESCRIPTION
**Pull request by armandcela**
SET-285
Signed-off-by: Armand Cela <armand.cela@unipart.io>

### CODE REVIEW

**Review by adamrpatrick — APPROVED** [review]

File: addons/edi/models/edi_record.py [hunk 1]
```diff
@@ -106,8 +106,9 @@ def clear_cache_prepare(self):
         EdiRecordType = self.env["edi.record.type"]
 
         ir_model = IrModel.search([("model", "=", self._name)])
-        edi_record_type = EdiRecordType.search([("model_id", "=", ir_model.id)])
-        return edi_record_type.clear_cache_prepare
+        edi_record_types = EdiRecordType.search([("model_id", "=", ir_model.id)])
+        clear_cache_prepare = sum(edi_record_types.mapped("clear_cache_prepare"))
```
Comment (kdwyer) [inline addons/edi/models/edi_record.py:110]: I think we agreed with @ws-mps that we should never have more than one record type per EDI record, so I don't think this is correct, unless there have been further discussions.

That's why we removed the dummy record types in https://github.com/unipartdigital/odoo-edi/pull/51 (awaiting merge)

Comment (armandcela) [reply addons/edi/models/edi_record.py:110]: It was raising error without this change. Thought was related with setup in collingwood, that is for multi companies. Will have a look though

Comment (kdwyer) [reply addons/edi/models/edi_record.py:110]: Did you build your changes over #51 ?

Comment (armandcela) [reply addons/edi/models/edi_record.py:110]: I think you are right it doesnt have to do with multicompanies, there are 2 dummy record types in Collingwood. Will make changes there and most possible will close this MR.
Thank you

Comment (armandcela) [reply addons/edi/models/edi_record.py:110]: I am closing this MR, as changes are already done in https://gitlab.unipart.digital/udes/collingwood/-/merge_requests/25/diffs

**Comment by armandcela** [comment]
Closing MR as after https://gitlab.unipart.digital/udes/collingwood/-/merge_requests/25/diffs will be merged there wont be any issues as edi record type will always be single record

**Pull request closed without merging**